src/main/java/com/exasol/adapter/document/UdfCallBuilder.java

```diff
@@ -42,19 +42,31 @@
     private static final String REMOTE_TABLE_QUERY_PARAMETER = "REMOTE_TABLE_QUERY";
     private static final String CONNECTION_NAME_PARAMETER = "CONNECTION_NAME";
     private static final String FRAGMENT_ID = "FRAGMENT_ID";
+    private final String connectionName;
+    private final String adapterName;
+
+    /**
+     * Create an instance of {@link UdfCallBuilder}.
+     * 
+     * @param connectionName connectionName that is passed to the UDF
+     * @param adapterName    name of the adapter
+     */
+    public UdfCallBuilder(final String connectionName, final String adapterName) {
+        this.connectionName = connectionName;
+        this.adapterName = adapterName;
+    }
 
     /**
      * Build push down SQL statement with a UDF call to {@link ImportDocumentData}.
      * 
      * @param documentFetchers document fetchers. Each document fetcher gets a row that is passed to a UDF
```

Comment:
Please put the explanation in the description above, not in the parameter description.